benchmarks/distributed/rpc/ps/BenchmarkConfigurations.py

```diff
@@ -0,0 +1,31 @@
+class BenchmarkConfigurations:
```

Comment:
python dataclass could simplify this class (https://docs.python.org/3/library/dataclasses.html)

Reply:
okay I will use it.